### PR TITLE
Sw 53 bugfix consumerscorecheck

### PR DESCRIPTION
**Pull request by FATCHIP-GmbH**

### CODE REVIEW

**Comment by fjbender** [comment]
Also, some new issues were introduced: https://scrutinizer-ci.com/g/PAYONE-GmbH/shopware-5/inspections/241a6d74-876d-4756-bb05-a20f38543079/issues/

These mostly are concerned with loose comparison of strings and booleans; and with the missing discrimination between success and error execution paths. Is there any sensible way to do this more elegantly?

File: Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/AddressCheck.php [hunk 1]
```diff
@@ -74,7 +74,14 @@ public function check(Payone_Api_Request_AddressCheck $request)
             $response = $this->getMapperResponse()->map($responseRaw);
 
             $this->protocol($request, $response);
-        } catch (Exception $e) {
+
+            // throw exception for testing of connection failures
```
Comment (Fatchip-Support) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/AddressCheck.php]: shall i remove this completely or should i add a check that this function is only available for Mode= Testing?

Comment (fjbender) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/AddressCheck.php]: Yup, please add that condition :)

Comment (nether-cat) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/AddressCheck.php]: Done :)

File: Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/Consumerscore.php [hunk 2]
```diff
@@ -73,11 +73,16 @@ public function score(Payone_Api_Request_Consumerscore $request)
             $response = $this->getMapperResponse()->map($responseRaw);
 
             $this->protocol($request, $response);
-        } catch (Exception $e) {
+            
+            // throw exception for testing of connection failures 
+            if ($requestParams['firstname'] === 'Doris' && $requestParams['lastname'] === 'Beckmann'
```
Comment (fjbender) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/Consumerscore.php]: `$isLiveMode` condition is missing here, isn't it?

**Pull request closed without merging**